### ISSUE TITLE
Creating view results in syntax errors

### ISSUE THREAD

**Issue by qoelet** [sbdchd/squawk#473]
The following migration fails with `syntax-error` when run with Squawk 2.4.0,

```sql
create or replace view my_view as
select x from foo;
```

```
warning[syntax-error]: expected FUNCTION_KW
 --> bad.sql:1:18
  |
1 | create or replace view my_view as
  |                  ^
  |
warning[syntax-error]: expected L_PAREN
 --> bad.sql:1:23
  |
1 | create or replace view my_view as
  |                       ^
  |
warning[syntax-error]: expected function option
 --> bad.sql:1:23
  |
1 | create or replace view my_view as
  |                       ^
  |
warning[syntax-error]: expected SEMICOLON
 --> bad.sql:1:23
  |
1 | create or replace view my_view as
  |                       ^
  |
warning[syntax-error]: expected command, found IDENT
 --> bad.sql:1:24
  |
1 | create or replace view my_view as
  |                        ^
  |
warning[syntax-error]: expected command, found AS_KW
 --> bad.sql:1:32
  |
1 | create or replace view my_view as
  |                                ^
  |

Find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
Found 6 issues in 1 file (checked 1 source file)
```

**Comment by sbdchd** [comment]
Thank you for the report! Fixed in version 2.5.0!
